tests/pass/shims/fs.rs

```diff
@@ -4,12 +4,15 @@
 #![feature(io_error_more)]
 #![feature(io_error_uncategorized)]
 
+#[cfg(not(target_os = "freebsd"))]
 use std::collections::HashMap;
+#[cfg(not(target_os = "freebsd"))]
 use std::ffi::OsString;
```

Comment:
Please move these imports to inside `test_directory` rather than having such global `cfg`